### DISCUSSION TITLE
How can I identify an archived post so I can exclude them from my feed generator?

### DISCUSSION THREAD

**Discussion by seanthegeek** [bluesky-social/atproto#3204] [Bluesky Lexicons]
I have a feed that is based on keywords. When someone imports their old posts from Twitter/X, old content ends up in the feed. How can I identify an archived post so I can exclude them from my feed generator?

**Comment by DavidBuchanan314** [comment]
You can look at the `createdAt` timestamp. If it's in the past (with some margin for error, not everyone's system clock is accurate), then it's probably archive import.
